sp800-63b/sec3_definitions.md

```diff
@@ -147,9 +147,21 @@ A set of attributes that uniquely describe a person within a given context.
 A metric describing degree of confidence that the Applicant’s Claimed Identity is their real identity.
 
 #### Kerberos
```

Comment:
Kerberos is not referenced in SP 800-63B, so this definition probably belongs in -63C instead, where the term is used.

The definition is too long already; it should just describe quickly what it is in a sentence or two. The description of how it works belongs in -63C Section 11.2.